src/portable/espressif/esp32s2/dcd_esp32s2.c

```diff
@@ -167,15 +167,22 @@ void dcd_init(uint8_t rhport)
 {
   ESP_LOGV(TAG, "DCD init - Start");
 
-  // A. Disconnect
-  ESP_LOGV(TAG, "DCD init - Soft DISCONNECT and Setting up");
-  USB0.dctl |= USB_SFTDISCON_M; // Soft disconnect
+  bool did_persist = (USB_WRAP.date.val & (1 << 31)) != 0;
```

Comment:
Could you put an few line of description on what is the persist feature and how it work e.g how to enable persist mode. This feature is not obvious with others.

Reply:
@hathach I added some description